### ISSUE TITLE
libsswjni.so documentation isn't clear

### ISSUE THREAD

**Issue by datakid** [PapenfussLab/gridss#37]
1. It should be mentioned as a pre-requisite or "strongly recommended" near the top where the docs mention "running".
2. The paragraph itself isn't clear. Needs to explicitly mention that a _wrapper_ is provided, and that the SSW needs to be downloaded and compiled. 
3. Unless I am correct in thinking you are actually providing a precompiled copy of SSW in the gridss jar - which is my preferred interpretation of the section:

`GRIDSS will attempt to load a precompiled version. If the precompiled version is not compatible with your linux distribution, or you are running a different operating system, recompilation of the wrapper from source will be required.`

The phrases "your linux distribution" and "different operating system" are not clarified at any point, making them poor measures. 

Some of my researchers are asking me if it's installed, not being sure if your paragraph is referring to the gridss provided SSW wrapper or SSW itself.

**Comment by datakid** [comment]
The next part of this issue is:
 - how can I tell if SSW is being used?

**Comment by d-cameron** [comment]
ilibsswjni.so is indeed packaged in the GRIDSS jar and is unloaded to TMP_DIR when running. I have clarified this in the README.

When GRIDSS starts, there is a INFO message stating "Loading aligner". If libsswjni does not load successfully. then the next line will be an ERROR message stating "Unable to use sswjni library - assembly will be very slow.

> The phrases "your linux distribution" and "different operating system" are not clarified at any point, making them poor measures.

They are indeed, and unfortunately, that is nature of C compilations. Fortunately, the SSW library itself has no library dependencies which greatly reduces the opportunity for an ABI mismatch. I haven't come across a linux distribution that fails to load the library, but that doesn't mean there aren't ones out that that will fail. I have encountered failure running on old CPUs that do not implement the required SSE instruction set, and have added documentation around that.

In summary, SSW does not features prominently in the installation instructions unless there is an error message about sswjni, there's nothing to do.

**Comment by datakid** [comment]
A1, thanks.
